app/views/govuk_publishing_components/components/_subscription-links.html.erb

```diff
@@ -1,21 +1,50 @@
 <%
   email_signup_link ||= false
+  email_signup_link_text ||= t("govuk_component.subscription_links.email_signup_link_text", default: "Get email alerts")
   feed_link ||= false
+  feed_link_text ||= t("govuk_component.subscription_links.feed_link_text", default: "Subscribe to feed")
+  feed_link_box_value ||= false
```

Comment:
This is way too late (again), but will `feed_link_box_value` ever be different from `feed_link`? Perhaps a boolean param to show the box or not would make it easier to use.